.github/workflows/unit-test.yml

```diff
@@ -165,6 +165,7 @@ jobs:
     runs-on: ubuntu-latest
     env:
       NPM_CONFIG_UNSAFE_PERM: true
+      NODE_OPTIONS: --max-old-space-size=4096
```

Comment:
On my machine, tests for `user-interactions` are failing with 2G of memory, but tests for `long-task` which has very similar dependencies are passing even with `256MB`.

I don't mind increasing the memory, but I think there is a real issue with the `user-interactions` code/deps/config which we are missing.

Reply:
As I mentioned in the issue, this is probably related to the code coverage plugin for webpack which consumes more memory due to 
https://www.npmjs.com/package/convert-source-map/v/1.9.0

I don't think we have a lot to do besides increasing the memory, but at least we know that it is not related to our code.